modelling/src/main/java/org/axonframework/modelling/entity/PolyMorphicEntityModelBuilder.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2010-2025. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.modelling.entity;
+
+import jakarta.annotation.Nonnull;
+import jakarta.annotation.Nullable;
+import org.axonframework.messaging.QualifiedName;
+import org.axonframework.modelling.EntityEvolver;
+import org.axonframework.modelling.entity.child.EntityChildModel;
+
+/**
+ * Builder for a polymorphic {@link EntityModel}, where a parent entity can have multiple concrete child entities.
+ * Command handlers of concrete types take precedence over the parent entity's command handlers. Event handlers are
+ * invoked on both, with the super entity's event handlers being invoked first.
+ *
+ * @param <E>
+ * @author Mitchell Herrijgers
+ * @see PolymorphicEntityModel
+ * @since 5.0.0
+ */
+public interface PolyMorphicEntityModelBuilder<E> extends EntityModelBuilder<E> {
```

Comment:
PolyMorphicEntityModelBuilder
```suggestion
public interface PolymorphicEntityModelBuilder<E> extends EntityModelBuilder<E> {
```

I belive Polymorphic is a single word. Example: `PolymorphicSerializer` from Kotlin.

Reply:
You are right, it is a typo. Adjusting!